src/modules/fw_rate_control/FixedwingRateControl.cpp

```diff
@@ -51,6 +51,8 @@ FixedwingRateControl::FixedwingRateControl(bool vtol) :
 	/* fetch initial parameter values */
 	parameters_update();
 
+	_handle_param_vt_fw_difthr_en = param_find("VT_FW_DIFTHR_EN");
```

Comment:
```suggestion
	_handle_param_vt_fw_difthr_en = param_find("VT_FW_DIFTHR_EN");
	parameters_update();
```
No? Othwrwise it's invalid during the initial parameter fetch.

Reply:
yes sure, done.